.github/workflows/build.yml

```diff
@@ -141,3 +144,11 @@ jobs:
         with:
           name: build-scan-test-${{ matrix.java }}
           path: ~/.gradle/build-scan-data
```

Comment:
(this comment is unrelated to this PR)

Should we add `compression-level: 9` to this archive as all of them are plaintext.